lib/net/ssh/multi/session.rb

```diff
@@ -447,7 +450,7 @@ def io_select_wait(wait)
 
     def keepalive_interval
       servers = server_list.select { |s| s.options[:keepalive] }
-      servers.map { |s| s.options[:keepalive_interval] }.compact.min
+      servers.map { |s| s.options[:keepalive_interval] || DEFAULT_IO_SELECT_TIMEOUT }.compact.min
```

Comment:
i'd rather add somthing to net-ssh, there seems to be too much logic duplication. Somthing like this:
```ruby
servers.map { |s| s.io_select_wait }.compact.min
```